release-notes/dashboards-assistant.release-notes-2.12.0.0.md

```diff
@@ -0,0 +1,15 @@
+## Version 2.12.0.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.12.0
+
+### Features
```

Comment:
Shall we call it `Experimental Features`? I remember we need to mark it as experimental, am I right? @wanglam

Reply:
Yeah. Since our plugin is experimental, we should use experimental features like [ml-commons-dashboards 2.6](https://github.com/opensearch-project/ml-commons-dashboards/blob/main/release-notes/opensearch-ml-commons-dashboards.release-notes-2.6.0.0.md)